### PR TITLE
MGDOBR-1210: Review manager-xxx application.properties

### PR DESCRIPTION
**Pull request by manstis**
[https://issues.redhat.com/browse/MGDOBR-1210](https://issues.redhat.com/browse/MGDOBR-1210)

I ran the whole stack locally (`./dev/bin` scripts) and successfully created a `Bridge`.

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: manager-parent/manager-core/src/main/resources/application.properties [hunk 1]
```diff
@@ -178,4 +109,46 @@ event-bridge.processing-errors.cleanup.schedule=0 0/5 * * * ? *
 
 # QUOTAS
 event-bridge.account-management-service.quota.config=${EVENT_BRIDGE_ACCOUNT_MANAGEMENT_SERVICE_QUOTA_CONFIG}
+
```
Comment (manstis) [inline manager-parent/manager-core/src/main/resources/application.properties:112]: I moved all the required `DEV` overrides into one block.

File: manager-parent/manager-core/src/test/resources/application.properties [hunk 2]
```diff
@@ -37,4 +37,8 @@ event-bridge.resources.workers.job-retry-interval-seconds=1
 # Disable error processing scheduler
 event-bridge.processing-errors.cleanup.schedule=0 0 0 1 1 ? 2099
 
+# LOGGING
+# Use unstructured logging in test
+quarkus.log.console.json=false
```
Comment (manstis) [inline manager-parent/manager-core/src/test/resources/application.properties:42]: I moved the `TEST` override to `src/main/test/resources/application.properties` for consistency with other test overrides.

File: manager-parent/manager-core/src/main/resources/application.properties [hunk 3]
```diff
@@ -153,9 +87,6 @@ event-bridge.dns.aws.route53.secret-access-key=${AWS_SECRET_ACCESS_KEY}
 event-bridge.dns.ingress.override.hostname=${EVENT_BRIDGE_DNS_INGRESS_OVERRIDE_HOSTNAME:localhost}
 
 # LOGGING
-# Use unstructured logging in DEV/test
-%dev.quarkus.log.console.json=false
-%test.quarkus.log.console.json=false
```
Comment (manstis) [inline manager-parent/manager-core/src/main/resources/application.properties:158]: I moved the `TEST` override to `src/main/test/resources/application.properties` for consistency with other test overrides.

File: manager-parent/manager-core/src/main/resources/application.properties [hunk 4]
```diff
@@ -10,19 +10,6 @@ rhose.metrics-name.operation-duration-seconds=managed_services_api_rhose_operati
...
-quarkus.smallrye-openapi.store-schema-directory=target/generated/
-quarkus.smallrye-openapi.info-title=Red Hat Openshift SmartEvents Fleet Manager
-quarkus.smallrye-openapi.info-version=0.0.1
-quarkus.smallrye-openapi.info-description=The API exposed by the fleet manager of the SmartEvents service.
-quarkus.smallrye-openapi.info-license-name=Apache 2.0
-quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0
-quarkus.smallrye-openapi.info-contact-email=openbridge-dev@redhat.com
-quarkus.smallrye-openapi.info-contact-name=Development Team
-quarkus.smallrye-openapi.servers=https://api.openshift.com,https://api.stage.openshift.com,http://localhost:8000,/
-mp.openapi.extensions.smallrye.operationIdStrategy=CLASS_METHOD
```
Comment (r00ta) [inline manager-parent/manager-core/src/main/resources/application.properties:24]: I think I'm missing something. Do we have this config somewhere else?

Comment (manstis) [reply manager-parent/manager-core/src/main/resources/application.properties:24]: No.. I meant to add it to the `manager-v1` properties.. but, based on the failing build, I forgot.

Comment (manstis) [reply manager-parent/manager-core/src/main/resources/application.properties:24]: I guess, it'd be safer keeping it on `manager-core`.... so I'll add it back.

Comment (manstis) [reply manager-parent/manager-core/src/main/resources/application.properties]: I also added an _explicit_ dependency for `io.quarkus:quarkus-smallrye-openapi`.

**Review by r00ta — APPROVED** [review]

**Review by danielezonca — APPROVED** [review]

**Review by ChughShilpa — APPROVED** [review]

**Pull request merged**